src/common/snippets/src/pass/fq_decomposition.cpp

```diff
@@ -205,48 +207,100 @@ bool ngraph::snippets::pass::FakeQuantizeDecomposition::getScalesAndShifts(
     if (!input_low_constant || !input_high_constant || !output_low_constant || !output_high_constant)
         return false;
 
+    const auto input_low_pshape = input_low_constant->get_output_partial_shape(0);
+    const auto input_high_pshape = input_high_constant->get_output_partial_shape(0);
+    const auto output_low_pshape = output_low_constant->get_output_partial_shape(0);
+    const auto output_high_pshape = output_high_constant->get_output_partial_shape(0);
```

Comment:
These nodes are Constants so we can safely get their `Shape` instead of `PartialShape` and avoid additional conversions below
```suggestion
    const auto input_low_shape = input_low_constant->get_shape();
    const auto input_high_shape = input_high_constant->get_shape();
    const auto output_low_shape = output_low_constant->get_shape();
    const auto output_high_shape = output_high_constant->get_shape();
```

Reply:
Sure, done. Thanks!